### PR TITLE
SkyeKiwi Application

### PR DESCRIPTION
**Pull request by RoyTimes**
I have read and agree with the [T&C](https://github.com/crustio/Crust-Grants-Program/blob/main/crust-grants-terms-and-conditions.md) from the repo. All software outlined in the application will be released under Apache 2.0 Opensource License. An ETH address is provided in the application and we opt to receive USDT. 

I'll try to respond to any question/comment within 2 days. Thanks for reviewing.

### CODE REVIEW

**Comment by zikunfan** [comment]
lgtm, already proposed a meet with team, looking forward to talk about `SkyPass` later. 😄

**Comment by zikunfan** [comment]
Talked offline, very cool project. We can adjust the application to more fit to Crust Network, think about `encrypto-NFT` or `encrypto-IPFS-tool`

**Comment by RoyTimes** [comment]
@zikunfan We have updated the application per our previous discussion. Let us know if you have any questions.

File: applications/skyepass.md [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+  
+  
+
+| Number | Deliverable           | Specification                                                |
+| ------ | --------------------- | ------------------------------------------------------------ |
+| 0a.    | License               | Apache 2.0 on all repos                                      |
+| 0b.    | Documentations        | A guideline of how to run and test all functionalities described below. |
+| 1.     | Sample Smart Contract | The core smart contract that stores IPFS hash, generate unique vault ID and implement access management. <br/>We are using ink! and the smart contract development suite maintained and developed by [Patract Labs](https://patract.io/) for developing environment, unit testing and deployment. |
+| 2.     | Protocol Library      | 1. 3 sample encryption schema as described above and tooling to build customized encryption schema. <br/>2. Core metadata handler that takes in a file and the encryption schema to generate a sharable metadata file. <br/>3. Use a sample smart contract to facilitate authorization management and file updating. <br/>4. Persistent storage with Crust Network APIs<br/>5. Unit tests and integration test w/Chia |
```
Comment (zikunfan) [inline applications/skyepass.md:211]: what's this *chia* means? Is that the blockchain chia or a testing framework?

**Review by zikunfan — CHANGES_REQUESTED** [review]

File: applications/skyepass.md (hunk 1 above)
Comment (RoyTimes) [reply applications/skyepass.md:211]: the testing framework on node.js

**Comment by RoyTimes** [comment]
Updates:

1. Medium posts is now included in deliveries
2. I have removed the part on multi-party decryption, instead, replaced it with a bunch of demos along with the post to help developers better use the protocol. 
3. I have updated the amount of grant requested

**Review by zikunfan — APPROVED** [review]
LGTM

**Review by yyd106 — APPROVED** [review]

**Pull request merged**